docs/docs/guides/operate/graphql/graphql-client.md

```diff
@@ -27,25 +27,25 @@ Note that all GraphQL methods on the API are not yet available in Python - the `
   module="dagster_graphql"
   object="DagsterGraphQLClient"
   method="submit_job_execution"
-  />
+  /> [submit_job_execution](#submitting-a-job-run)
 - <PyObject
```

Comment:
These aren't showing the method names because the `method` prop isn't implemented in new docs -- you can fix these by appending the object name to the module and changing `method` to `module`: `<PyObject section="libraries" module="dagster_graphql.DagsterGraphQLCient" object="get_run_status" />`

Reply:
Ah that works to get the method name to display, but unfortunately then the hyperlink that gets generated for it is invalid and will 404. e.g. https://docs.dagster.io/api/python-api/libraries/dagster-graphql.DagsterGraphQLCient#dagster_graphql.DagsterGraphQLCient.submit_job_execution

Reply:
@neverett I think let's press ahead with this change as a stopgap